src/main/resources/database/schema_lockrepository.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+CREATE SCHEMA integration;
+SET search_path TO integration;
```

Comment:
Do you know what the impact exactly is for this?
It seems quite dangerous to me when I read some documentation.
Also it does not seem to be required for the int_lock..

Reply:
well the intention was to create the `sequence` in the integration database. so it was missing at the end of the script 
`SET search_path TO public;` just to leave things as it were. But for now you are right is not needed. Removed.